lib/ojo.pm

```diff
@@ -164,6 +166,15 @@ Encode Perl data structure or decode JSON with L<Mojo::JSON/"j">.
 
   $ perl -Mojo -E 'f("hello.json")->spurt(j {hello => "world!"})'
 
+=head2 l
+
+  my $url = l('https://mojolicious.org');
+
+Turn a string into a L<Mojo::URL> object.
+
+  $ perl -Mojo -E 'my $l = l "https://mojolicious.org";
+      say $l->clone->path($_) for c(split $/ => f("paths.txt")->slurp)->each'
```

Comment:
Do you really not have a nice one-liner example to demonstrate its usefulness?

Reply:
This PR was for #1514 and I could have different example: 

`perl -Mojo -E 'say l("/perldoc")->to_abs(l("https://mojolicious.org"))`

Reply:
I forgot that you do need a Mojo::URL for to_abs, that's a good case. I was tinkering with something like a file with a list of urls and getting all the hosts out of it

```
perl -Mojo -E 'print r(b(f(shift)->slurp)->split("\n")->map(sub{ l($_)->host })->uniq->to_array)' urls
```